src/app/digital/models/sim/Propagators.ts

```diff
@@ -60,6 +60,12 @@ export const AllPropagators: PropagatorRecord = {
     "LED": Noprop,
 
     "ANDGate": InputOutputPropagator((inputs) => [inputs.reduce(AND)]),
+
+    // ToDo: Add prop here later
+    "DFlipFlop":  Noprop,
+    "TFlipFlop":  Noprop,
+    "JKFlipFlop": Noprop,
+    "SRFlipFlop": Noprop,
```

Comment:
Still needs to be done

Reply:
Should we have all the propagator functions in a separate file

Reply:
Mmmmmm, yeah, sure. You should make a `propagators` subfolders in `src/app/digital/models/sim/` and have all the FlipFlop propagators in a `FlipFlops.ts` file within that folder